.github/workflows/publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish to PyPI.org
+on:
+  release:
+    types: [published]
+jobs:
+  pypi-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: python3 -m pip install --upgrade build && python3 -m build
+      - name: Publish package
+        uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
@mshudrak this introduces third-party code but since it is from pypa that we publish to it's probably fairly low risk.

Reply:
Yea, I think it is fine given it is well known org.

Reply:
yes basically, everybody uses this action.